packages/dbs-ai/client/views/app/tabbar/smarti.js

```diff
@@ -72,15 +73,14 @@ Template.dbsAI_smarti.helpers({
 /**
  * Load Smarti script
  */
-RocketChat.settings.onload('DBS_AI_Redlink_URL',function(key,value){
+RocketChat.settings.onload('DBS_AI_Redlink_URL', function() {
 
-	var DBS_AI_Redlink_URL = value.endsWith('/') ? value : value + '/';
+	Meteor.call('getSmartiUiScript', function(error, script) {
+		if (error) {
+			console.error('could not load Smarti:', error);
+		}
 
-	$.getScript(DBS_AI_Redlink_URL + 'plugin/v1/rocket.chat.js')
-		.done(function(){
-			console.debug('loaded Smarti successfully');
-		})
-		.fail(function( jqxhr, settings, exception ) {
-			console.error('could not load Smarti:', exception);
-		});
+		eval(script);
```

Comment:
This is a potential vulnerability since the script here is loaded from an external source. If somebody is able to change `DBS_AI_Redlink_URL` or `RocketChat.settings.get('DBS_AI_Redlink_URL')` an script can be injected.